src/background.ts

```diff
@@ -0,0 +1,103 @@
+"use strict";
+
+import { app, protocol, BrowserWindow } from "electron";
+import {
+    createProtocol,
+    // this file is mostly auto generated and imported into another generated project
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    installVueDevtools
+} from "vue-cli-plugin-electron-builder/lib";
+import path from "path";
+
+// hack to get __static working
+// see: https://github.com/electron-userland/electron-webpack/issues/172
+declare const __static: string;
+
+const isDevelopment = process.env.NODE_ENV !== "production";
+
+// Keep a global reference of the window object, if you don't, the window will
+// be closed automatically when the JavaScript object is garbage collected.
+let win: BrowserWindow | null;
+
+// Scheme must be registered before the app is ready
+protocol.registerSchemesAsPrivileged([
+    { scheme: "app", privileges: { secure: true, standard: true } }
```

Comment:
Why do we register a scheme as `app://` ?

Maybe `mhw://` or `myhbarwallet://` ?